src/components/Main/Main.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
```

Comment:
Consider using an interface instead of a type alias for defining component props.